qiita_pet/handlers/study_handlers/sample_template.py

```diff
@@ -57,3 +61,50 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown sample template action: %s' % action)
         self.write(result)
+
+
+def build_sample_summary(study_id, user_id, visible=None):
```

Comment:
This is the overall sample summary page @rob-knight and @antgonza asked for, where each sample is listed as part of or not part of a prep template's data set. The mockup is [here](https://biocore.mybalsamiq.com/projects/qiitastudiesv3/mouse%20decomp%20-%20sample%20summary).